### PR TITLE
Boot up arbitrum balance updater

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-1643

## Context
Test PR for discussion purposes regarding per-chain balance queries' routing

### CODE REVIEW

File: packages/backend/src/modules/tvl/ArbitrumTvl.ts [hunk 1]
```diff
@@ -38,13 +52,30 @@ export function createArbitrumTvlSubmodule(
...
+    multicallClient,
+  )
+
+  const arbitrumBalanceUpdater = new BalanceUpdater(
+    arbitrumBalanceProvider,
+    arbiscanBlockNumberUpdater,
+    db.balanceRepository,
+    db.balanceStatusRepository,
+    clock,
+    config.projects,
```
Comment (antooni) [inline packages/backend/src/modules/tvl/ArbitrumTvl.ts:66]: make this empty array for now

Comment (antooni) [reply packages/backend/src/modules/tvl/ArbitrumTvl.ts:66]: in the future we will come up wit a per-chain config

Comment (sdlyy) [reply packages/backend/src/modules/tvl/ArbitrumTvl.ts]: https://github.com/l2beat/l2beat/pull/1657/commits/937425eb298ac32c49d6068ebad8f90ac7a522b6

**Review by antooni — CHANGES_REQUESTED** [review]
please update the logs to include chainId, try to make it consistent with what we do at BlockNumberUpdater

image

**Review by antooni — APPROVED** [review]

**Pull request merged**